### PR TITLE
Eclair v0.10.0 release

### PR DESCRIPTION
**Pull request by t-bast**

### CODE REVIEW

**Review by sstone — CHANGES_REQUESTED** [review]
Just a few doc nits

File: docs/release-notes/eclair-v0.10.0.md [hunk 1]
```diff
@@ -0,0 +1,276 @@
...
+Without package relay, our anchor transactions may not propagate to miners, and eclair may end up bumping the fees more than the actual feerate, because it cannot know why the transaction isn't confirming.
+
+We introduced a new parameter to control the maximum fee that will be paid during fee-bumping, that node operators may configure:
+
+```eclair.conf
+// maximum amount of fees we will pay to bump an anchor output when we have no HTLC at risk
+eclair.on-chain-fees.anchor-without-htlcs-max-fee-satoshis = 10000
+```
+
+We also limit the feerate used to be at most the same order of magnitude of `fatest` feerate provided by our fee estimator.
```
[the remark this answers is not in the document]
Comment (t-bast) [reply docs/release-notes/eclair-v0.10.0.md:69]: What do you mean by "strange"? What do you suggest? I don't see a much better way of communicating this...

[the remark this answers is not in the document]
Comment (t-bast) [reply docs/release-notes/eclair-v0.10.0.md:69]: The logic I'm trying to communicate is that we always do: `targetFeerate.min(fastestFeerate * 1.25)`

Comment (sstone) [inline docs/release-notes/eclair-v0.10.0.md:69]:
```suggestion
We also limit the feerate used to be at most of the same order of magnitude as the `fastest` feerate provided by our fee estimator.
```

Comment (t-bast) [reply docs/release-notes/eclair-v0.10.0.md]: Done in https://github.com/ACINQ/eclair/pull/2831/commits/8772d9657e2aca4a8cad8a31d5d4dcb1fe6a8b8b

File: docs/release-notes/eclair-v0.10.0.md [hunk 2]
```diff
@@ -0,0 +1,276 @@
...
+
+### Managing Bitcoin Core wallet keys
+
+You can now use Eclair to manage the private keys for on-chain funds monitored by a Bitcoin Core watch-only wallet.
+
+See `docs/BitcoinCoreKeys.md` for more details.
```
[the remark this answers is not in the document]
Comment (t-bast) [reply docs/release-notes/eclair-v0.10.0.md]: Done in https://github.com/ACINQ/eclair/pull/2831/commits/8772d9657e2aca4a8cad8a31d5d4dcb1fe6a8b8b

**Review by sstone — APPROVED** [review]

File: docs/release-notes/eclair-v0.10.0.md [hunk 3]
```diff
@@ -0,0 +1,276 @@
+# Eclair vnext
```
[the remark this answers is not in the document]
Comment (t-bast) [reply docs/release-notes/eclair-v0.10.0.md:1]: Damn, good catch, thanks!

[the remark this answers is not in the document]
Comment (t-bast) [reply docs/release-notes/eclair-v0.10.0.md:1]: Will update in https://github.com/ACINQ/eclair/pull/2832

**Pull request merged**